src/js/components/showdogs/dog.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import Route from '../router/route';
+
+const pugPic = require('../../../assets/busci.jpg');
```

Comment:
Do not mix ES modules (import) with CJS modules (require).
Import this image instead.

Reply:
Thanks, done.